travis/utils.sh

```diff
@@ -37,28 +37,20 @@ install_cli() (
 install_appmap_java() (
   [[ -v DEBUG ]] && set -x
   set -o pipefail
-  APPMAP_JAVA_VERSION=java8
-  JAVAP_VERSION=$(javap -version | cut -d. -f1,2)
-  [[ "$JAVAP_VERSION" == 11.0 ]] && APPMAP_JAVA_VERSION=java11
+  APPMAP_JAR_DOWNLOAD_URL="$(curl -fsSL https://api.github.com/repos/applandinc/appmap-java/releases/latest \
+    | jq -e -r '.assets[0] | select(.name).browser_download_url')"
 
-  APPMAP_JAR_DOWNLOAD_URL="$(curl $(auth_arg) -fsSL https://api.github.com/repos/applandinc/appmap-java/releases/latest \
-    | jq -e --arg java_version $APPMAP_JAVA_VERSION  -r '.assets[] | select(.name | contains($java_version)).browser_download_url')"
-  if [[ $? != 0 ]]; then
```

Comment:
I think this check should still be here. The error message could be `Failed to find download URL for appmap jar`.